docs/plugins/testing_workshop_docs/Testing-Resources.md

```diff
@@ -0,0 +1,12 @@
+[Today’s plugin](https://github.com/DragaDoncila/plugin-tests) - Plugin Tests
```

Comment:
I think this stuff is all ripe for moving to the index page

Reply:
Wherever this goes, I suggest making it into a bulleted list for clearer formatting.

Reply:
Done.